ci/install-riff-system.yml

```diff
@@ -0,0 +1,26 @@
+steps:
+  - template: install-duffle-step.yml
+
+  - bash: ./git-repo-fetch.sh $(fatsDir) $(fatsRefspec) projectriff/fats
+    workingDirectory: '$(system.defaultWorkingDirectory)'
+    displayName: 'fetch fats'
+
+  - bash: |
+      $(fatsDir)/install.sh kubectl
+      $(fatsDir)/install.sh minikube
+      $(fatsDir)/install.sh gcloud
+    displayName: 'install tools'
+    env:
+      GCLOUD_CLIENT_SECRET: '$(GcloudClientSecret)'
+
+  - template: setup-minikube.yml
+
+  - bash: $(fatsDir)/start.sh
```

Comment:
installing riff-system should depend on an existing cluster, not be responsible for creating a new one.